src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,64 @@
+import "./Footer.css";
+function Footer() {
+  return (
+    <footer className="footer bg-style">
+      {/* <!-- Button Footer Row --> */}
+
+      <div className="button-footer">
+        <a src="">
+          <div className="logo"></div>
+        </a>
+        <button>
+          <i className="material-icons md-dark md-18">language</i>
+          <span className="footer-btn">
+            Language: <strong>English</strong>
+          </span>
+          <i className="material-icons md-dark md-18">arrow_drop_down</i>
+        </button>
+        <button>
+          <span className="footer-btn">
+            Country: <strong>Worldwide</strong>
+          </span>
+          <i className="material-icons md-dark md-18">arrow_drop_down</i>
+        </button>
+        <button>
+          <span className="footer-btn">
+            Restricted Mode: <strong>Off</strong>
+          </span>
+          <i className="material-icons md-dark md-18">arrow_drop_down</i>
+        </button>
+        <button>
+          <i className="material-icons md-dark md-18">history</i>
+          <span>History</span>
+        </button>
+        <button>
+          <i className="material-icons md-dark md-18">help</i>
+          <span>Help</span>
+        </button>
+      </div>
+
+      {/* <!-- Primary Links Footer --> */}
+
+      <ul className="primary-links-footer">
+        <li>About</li>
+        <li>Press</li>
+        <li>Copyright</li>
+        <li>Creators</li>
+        <li>Advertise</li>
+        <li>Developers</li>
+        <li>+YouTube</li>
```

Comment:
instead of writing it multiple time, can make array of list and map over them

Reply:
Yeah can have done that . Didn't thinked doing it like wise thanks for reviewing and also for the suggestion I will make it as suggested.